contracts/0.8.9/WithdrawalQueueERC721.sol

```diff
@@ -191,8 +193,8 @@ contract WithdrawalQueueERC721 is IERC721Metadata, WithdrawalQueue {
         delete _getTokenApprovals()[_requestId];
         request.owner = payable(_to);
 
-        _getRequestsByOwner()[_to].add(_requestId);
         _getRequestsByOwner()[_from].remove(_requestId);
+        _getRequestsByOwner()[_to].add(_requestId);
```

Comment:
as a kind of safety measure, I would suggest checking the returns of `add` and `remove` methods of the OZ `EnumerableSet` here and there, and maybe assert on them

Reply:
I think we can. It will be the third ring of defence, but relatively cheap one.